src/vfs/extfs/helpers/s3+.in

```diff
@@ -14,102 +14,49 @@
 #  GNU General Public License for more details.
 #
 #  You should have received a copy of the GNU General Public License
-#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
-#
-# Notes:
-#  This EXTFS exposes buckets as directories and keys as files
-#  Due to EXTFS limitations all buckets & keys have to be read initially which might
-#  take quite some time.
-#  Tested on Debian with Python 2.4-2.6 and boto 1.4c and 1.6b
-#    (Python 2.6 might need -W ignore::DeprecationWarning due to boto using
-#    deprecated module Popen2)
-#
-#
-# Installation:
-#  Make sure that boto <https://github.com/boto/boto> (python-boto in Debian) is installed.
-#  Preferably pytz (package python-tz in Debian) should be installed as well.
-#
-#  Save as executable file /usr/libexec/mc/extfs/s3 (or wherever your mc expects to find extfs modules)
-#
-# Settings: (should be set via environment)
-#  Required:
-#    AWS_ACCESS_KEY_ID         : Amazon AWS access key (required)
-#    AWS_SECRET_ACCESS_KEY     : Amazon AWS secret access key (required)
-#  Optional:
-#    MCVFS_EXTFS_S3_LOCATION   : where to create new buckets: "EU" - default, "USWest", "APNortheast" etc.
-#    MCVFS_EXTFS_S3_DEBUGFILE  : write debug info to this file (no info by default)
-#    MCVFS_EXTFS_S3_DEBUGLEVEL : debug messages level ("WARNING" - default, "DEBUG" - verbose)
```

Comment:
Could you please leave the settings block and update it to reflect the current state?